README.md

```diff
@@ -87,9 +87,9 @@ If you want, you can initialize install manifest directory with [anyenv/anyenv-i
 If you have own manifest repository, you can specify it:
 
 ```
-$ anyenv install --init https://github.com/foo/anyenv-install.git
+$ anyenv install --init https://github.com/anyenv/anyenv-install.git
```

Comment:
This is intentional since this option should be used by a custom manifests, not for default one.